doc/manuals/admin/build_source.md

```diff
@@ -145,7 +142,7 @@ sudo scons install --prefix=/usr/local --use-sasl-client --ssl  # This puts .h f
 
 ## Others
 
-If you have build orion in a system different from CentOS 6.x, don't hesitate to tell us and contribute to expand this section. Probably the best way if doing a pull request to modify this file with the new information. Thanks!
+If you have build orion in a system different from CentOS 7.x, don't hesitate to tell us and contribute to expand this section. Probably the best way if doing a pull request to modify this file with the new information. Thanks!
```

Comment:
build => built
the best way if doing => the best way **is to issue** a pull request ...

Reply:
Fixed in 952c2e2